src/main/java/SpaceFilter.java

```diff
@@ -0,0 +1,39 @@
+import java.util.ArrayList;
+
+/**
+ * This class represents a filter. It works by checking a given schedule for the given criteria.
+ * It is made "stackable" by returning the schedule if it works, and then that can be used for further calls.
+ */
+public class SpaceFilter implements Filter {
+    // This is the interval between classes that you want, in hours
+    private final int interval;
+
+    public SpaceFilter(int hours) {
```

Comment:
Put the comment from above `private final int interval` as the javadoc comment for this constructor basically